### ISSUE TITLE
apache_conf does not read symlinked configuration files

### ISSUE THREAD

**Issue by carldjohnston** [inspec/inspec#1401]
### Description

Symlinked configuration files are not read by the apache_conf resource. 

Using the apache2 cookbook to configure apache will create links to configuration files from `conf-enabled/configfile.conf` to `conf-available/configfile.conf`; the `conf-enabled/*.conf` directory is then included through an `IncludeOptional` apache directive. I believe this is the default behavior on Ubuntu also (without the apache2 cookbook).

### InSpec and Platform Version

InSpec 1.7.1 on Oracle Linux 7.3

### Replication Case

Apply the apache2::default recipe to a system, and check for directives that are included by default in `/etc/httpd/conf-enabled/security.conf` such as `ServerTokens Prod` with:

```
describe apache_conf do
  its('ServerTokens') { should cmp 'Prod' }
end
```

### Possible Solutions

Perhaps adding the following as line 110 to apache_conf.rb might solve the issue:
`files += find_files(id, depth: 1, type: 'link')`

https://github.com/chef/inspec/blob/master/lib/resources/apache_conf.rb#L109

**Comment by chris-rock** [comment]
Are you opening up a PR to fix that?

**Comment by adamleff** [comment]
Believe this is fixed in #1406, so I'm going to close this issue. Please reopen if that's not the case. Thanks!
